vendor/src/github.com/containers/image/manifest/manifest.go

```diff
@@ -0,0 +1,97 @@
+package manifest
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+	"encoding/json"
+
+	"github.com/docker/libtrust"
+	imgspecv1 "github.com/opencontainers/image-spec/specs-go/v1"
+)
+
+// FIXME: Should we just use docker/distribution and docker/docker implementations directly?
+
+// FIXME(runcom, mitr): should we havea mediatype pkg??
+const (
+	// DockerV2Schema1MIMEType MIME type represents Docker manifest schema 1
+	DockerV2Schema1MIMEType = "application/vnd.docker.distribution.manifest.v1+json"
```

Comment:
@stevvooe I'm making a patch but I'm not following why this is important in this PR review since those aren't even used :/